data/random-teams.ts

```diff
@@ -120,16 +120,12 @@ export class RandomTeams {
 				if (!counter.Dark) return true;
 				return hasMove['suckerpunch'] && (movePool.includes('knockoff') || movePool.includes('wickedblow'));
 			},
-			Dragon: (movePool, hasMove, hasAbility, hasType, counter, species) => {
-				if (movePool.includes('outrage') && species.baseStats.atk >= 115) return true;
-				return (
-					!(counter.setupType === 'Physical' && hasType['Flying']) &&
-					!counter.Dragon &&
-					!hasMove['dragonascent'] &&
-					!hasMove['substitute'] &&
-					!(hasMove['rest'] && hasMove['sleeptalk'])
-				);
-			},
+			Dragon: (movePool, hasMove, hasAbility, hasType, counter) => (
+				!counter.Dragon &&
```

Comment:
I don't fully understand the change in logic here, but the unit test for forcing Outrage still passes, so I guess it works. It looks like you're not forcing Flying STAB on Flying/Dragon types?

Reply:
yes, check the description.

your fix last update just added a hardcode to negate another hardcode for the same thing. i just removed both of them and made it simpler.

Reply:
OK.